src/components/user/uc/users.uc.unit.test.js

```diff
@@ -79,6 +114,16 @@ describe('users usecase', () => {
 
 		sinon.stub(accountRepo, 'deleteAccountForUserId');
 
+		getPseudonymsForUserStub = sinon.stub(pseudonymRepo, 'getPseudonymsForUser');
```

Comment:
since the users uc doesnt use the pseudonym repo directly, but rather the uc/facade, its sufficient to stup these.